mysql-test/suite/period/r/alter.result

```diff
@@ -0,0 +1,49 @@
+set @s= '1992-01-01';
+set @e= '1999-12-31';
+create or replace table t (s date, e date);
+alter table t add period for a(s, e);
+ERROR HY000: Period field `s` cannot be NULL
+alter table t change s s date, add period for a(s, e);
+ERROR HY000: Period field `e` cannot be NULL
+# NOT NULL is added implicitly if field is altered
+alter table t change s s date, change e e date, add period for a(s, e);
+show create table t;
+Table	Create Table
+t	CREATE TABLE `t` (
+  `s` date NOT NULL,
+  `e` date NOT NULL,
+  PERIOD FOR `a` (`s`, `e`)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1
+insert t values(@e, @s);
+ERROR 23000: CONSTRAINT `CONSTRAINT_1` failed for `test`.`t`
+alter table t add period for a(s, e);
+ERROR HY000: cannot specify more than one application-time period
+alter table t drop constraint constraint_1;
+ERROR HY000: Can't DROP CONSTRAINT `CONSTRAINT_1`. Use DROP PERIOD `a` for it
+alter table t drop period for a;
+# Constraint is dropped
+insert t values(@e, @s);
+alter table t drop period for a;
+ERROR 42000: Can't DROP PERIOD `a`; check that it exists
+alter table t add period for a(s, e), drop period for a;
+ERROR 42000: Can't DROP PERIOD `a`; check that it exists
+truncate t;
+alter table t add period for a(s, e);
+insert t values(@e, @s);
+ERROR 23000: CONSTRAINT `CONSTRAINT_1` failed for `test`.`t`
+alter table t add period for a(s, e), drop period for a;
+insert t values(@e, @s);
+ERROR 23000: CONSTRAINT `CONSTRAINT_1` failed for `test`.`t`
+alter table t add s1 date, add period for b(s1, e), drop period for a;
+show create table t;
+Table	Create Table
+t	CREATE TABLE `t` (
+  `s` date NOT NULL,
+  `e` date NOT NULL,
+  `s1` date NOT NULL,
+  PERIOD FOR `b` (`s1`, `e`)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1
+insert t(s, s1, e) values(@e, @s, @e);
+insert t(s, s1, e) values(@e, @e, @s);
+ERROR 23000: CONSTRAINT `CONSTRAINT_1` failed for `test`.`t`
+create or replace database test;
```

Comment:
## Bug 1: error text is misleading
### Reproduce
```sql
create or replace table t (a int, s date not null, e date not null);
alter table t add period for a(s, e);
```
### Result
```
ERROR 4149 (HY000): Could not specify name `a` for field. It is already used by period.
```
### Expected
```
Could not specify name `a` for period. It is already used by field.
```
The same error happens on `CREATE TABLE`. I guess the best way is to make universal text like:
```
Period and field `a` cannot have same name.
```